assets/scss/components/_button.scss

```diff
@@ -257,6 +258,17 @@ Class                   | Description
 	background: $C_contentBG;
 }
 
+.button--reset {
+	@include buttonHover(transparent);
+	background: transparent;
```

Comment:
Maybe we should file something separately to deal with the aggressive `.button[disabled]` rules